### PR TITLE
Topics: add three new topic pages

### PR DESCRIPTION
**Pull request by harding**
- CLTV expiry delta
- Proof of reserves
- X-only public keys

Topics were suggested on #721

### CODE REVIEW

File: _topics/en/cltv-expiry-delta.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+    Refund after expiry to Alice and payment to Carol:
+
+        Alice <------ Bob ------> Carol
+              (1 BTC)     (1 BTC)
+
+The CLTV expiry delta tries to prevent Bob from losing value this way.
+When Alice gives Bob an HTLC that allows her to claim a refund after
+`x` blocks, Bob gives Carol an HTLC that allows him to claim a refund
+after `x - y` blocks.  They _y_ parameter is Bob's HTLC expiry delta:
```
Comment (LarryRuane) [inline _topics/en/cltv-expiry-delta.md:126]:
```suggestion
after `x - y` blocks.  The _y_ parameter is Bob's HTLC expiry delta:
```

File: _topics/en/cltv-expiry-delta.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+to get an HTLC refund transaction confirmed onchain before that node is
+at risk of losing funds.  However, higher HTLC deltas magnify the
+problems of channel stalling, both accidental stalling (e.g. a node goes
+offline suddenly) and malicious stalling (e.g. [channel jamming
+attacks][topic channel jamming attacks]).
+
+For example, imagine a payment that will be sent across 20 hops each
+with an HTLC expiry delta of 100 blocks.  If that payment stalls, it
+could be up to 2,000 blocks (about two weeks) until the spender gets
+a refund and is able to resend the payment again.
```
Comment (LarryRuane) [inline _topics/en/cltv-expiry-delta.md:140]:
```suggestion
a refund and can resend the payment again.
```

File: _topics/en/cltv-expiry-delta.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+
+There's no universally agreed upon tradeoff between security and
```
Comment (LarryRuane) [inline _topics/en/cltv-expiry-delta.md:142]:
```suggestion
There's no universally agreed-upon tradeoff between security and
```

File: _topics/en/proof-of-reserves.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+deposited amounts if it was willing.
+
+Proof of reserves can not guarantee that the funds are available.  For
+example, Bank Corp could contract with a third party who controls 6 BTC
+to have them generate the proof.  In that case, Bank Corp may not
+control any bitcoins even though its "proof" seems to indicate that it
+does.
+
+The simplest form of proof of reserves lists each depositor's name in a
+form that can't be confused with any other depositor's name.  Explicit
```
Comment (LarryRuane) [inline _topics/en/proof-of-reserves.md:74]:
```suggestion
form that can't be confused with any other depositor's name.  The use of explicit
```

File: _topics/en/proof-of-reserves.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+names prevents the custodian from using the same bitcoins as proof for
+two or more different depositors.  For example, imagine two different
+people both named Alice who both have 1 BTC on deposit.  The example
+proof described for 6 BTC earlier would satisfy both of them, plus Bob
+and Carol, even though the actual total on deposit needs to be 7 BTC.
+
+For privacy, each depositor's name may be replaced with a pseudonym.
+However, depositors still need to ensure they are each given a unique
+name.  For more privacy, the association between name and exact deposit
```
Comment (LarryRuane) [inline _topics/en/proof-of-reserves.md:83]:
```suggestion
name.  For more privacy, the association between the name and the exact deposit
```

File: _topics/en/x-only-public-keys.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+do this, containing the bit within their prefix byte for a total of 33
+bytes.
+
+The extra bit can be eliminated if only one of the two alternative _y_
+coordinates are allowed.   X-only public keys do this, using a total of
+32 bytes.  Segwit v1 ([taproot][topic taproot]) and the initial version
+of [tapscript][topic tapscript] use x-only public keys for all signature
+checking operations to reduce the size of public keys.
+
+The advantage for x-only public keys is that they save space.  The
```
Comment (LarryRuane) [inline _topics/en/x-only-public-keys.md:94]:
```suggestion
The advantage of x-only public keys is that they save space.  The
```

File: _topics/en/x-only-public-keys.md [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+downside is that generated public keys most only used the allowed _y_
```
Comment (LarryRuane) [inline _topics/en/x-only-public-keys.md:95]:
```suggestion
downside is that generated public keys only used the allowed _y_
```
(I think? I may not be parsing this sentence correctly.)

File: _topics/en/cltv-expiry-delta.md [hunk 8]
```diff
@@ -0,0 +1,148 @@
...
+        Alice ------> Bob ------> Carol
+              (1 BTC)     (1 BTC)
+
+If Carol doesn't claim the payment by releasing the [HTLC][topic
+HTLC] preimage, but she also doesn't reject the payment, Bob's funds are
+stuck.  Not only that, but he can't resolve the forwarded payment he
+received from Alice, so her funds are stuck as well.  To avoid funds
+becoming permanently stuck, HTLCs have an expiry after which Bob will
+be able to claim a refund.  After Bob receives his refund from Carol, he
+can reject the payment from Alice (giving her a refund).  Alternatively,
```
Comment (LarryRuane) [inline _topics/en/cltv-expiry-delta.md:93]:
```suggestion
can reject the payment from Alice, giving her a refund.  Alternatively,
```
(a weak suggestion, just personal preference, feel free to ignore)

File: _topics/en/cltv-expiry-delta.md [hunk 9]
```diff
@@ -0,0 +1,148 @@
...
+
+For example, imagine a payment that will be sent across 20 hops each
+with an HTLC expiry delta of 100 blocks.  If that payment stalls, it
+could be up to 2,000 blocks (about two weeks) until the spender gets
+a refund and is able to resend the payment again.
+
+There's no universally agreed upon tradeoff between security and
+worst-case payment delivery time, so LN implementations tend to each use
+different default HTLC expiry deltas, often change those defaults, and
+usually allow users to chose their own setting.
```
Comment (LarryRuane) [inline _topics/en/cltv-expiry-delta.md:145]:
```suggestion
usually allow users to choose their own setting.
```

File: _topics/en/proof-of-reserves.md [hunk 10]
```diff
@@ -0,0 +1,103 @@
...
+---
+For example, Alice, Bob, and Carol all have bitcoin-denominated deposits
+with Bank Corp.  Each week, Bank Corp uses the private key controlling
+their onchain funds to sign a message saying "Alice has 1 BTC on
+deposit; Bob has 2 BTC on deposit; and Carol has 3 BTC on deposit."  If
+the key Bank Corp uses to sign that message contains 6 BTC or more, the
+depositors have some assurance that Bank Corp could return their
+deposited amounts if it was willing.
+
+Proof of reserves can not guarantee that the funds are available.  For
```
Comment (LarryRuane) [inline _topics/en/proof-of-reserves.md:67]:
```suggestion
Proof of reserves cannot guarantee that the funds are available.  For
```

File: _topics/en/proof-of-reserves.md [hunk 11]
```diff
@@ -0,0 +1,103 @@
...
+amount of proven reserves.  For example, Alice is shown that her
+contribution is 1 BTC and that the contribution of the amount paired
+with hers is 2 BTC.  That aggregated amount is paired with some number of
+accounts also totaling 3 BTC for a grand total of 6 BTC:
+
+                   6 BTC
+                    / \
+              3 BTC     3 BTC
+             /     \
+    (1 BTC: Alice)
```
Comment (LarryRuane) [inline _topics/en/proof-of-reserves.md:95]: I'm not sure, but does the node (hash) immediately to the right of Alice's node also need to be revealed? That is, both of the contributors to the "3 BTC" on the left side?

File: _topics/en/x-only-public-keys.md (hunk 7 above)
Comment (bitschmidty) [reply _topics/en/x-only-public-keys.md:95]: I have similar confusion on the sentence

**Comment by bitschmidty** [comment]
Doesnt seem like anything too intense coming out of these reviews (thanks @vostrnad and @LarryRuane !). You think we should work to get these fixups in and merge this PR, @harding ?

File: _topics/en/proof-of-reserves.md (hunk 11 above)
Comment (harding) [reply _topics/en/proof-of-reserves.md:95]: Updated to separate the sum operations from the name operations, so it's (maybe) a little clearer that all Alice needs is the clear-text amount of the side branch and the hash commitment to the name of the person with that balance.

File: _topics/en/cltv-expiry-delta.md [hunk 12]
```diff
@@ -0,0 +1,148 @@
...
+## Optional.  Each entry requires "title" and "url".  May also use "feature:
+## true" to bold entry and "date"
+optech_mentions:
+  - title: "BOLTs #1086 specifies 2,016 blocks as the default absolute maximum CLTV expiry"
+    url: /en/newsletters/2023/11/01/#bolts-1086
+
+  - title: "LND #2759 lowers the default CLTV delta for all channels from 144 blocks to 40 blocks"
+    url: /en/newsletters/2019/04/02/#lnd-2759
+
+  - title: "LN-Symmetry requires longer HTLC expiry deltas than expected"
```
[the remark this answers is not in the document]
Comment (harding) [reply _topics/en/cltv-expiry-delta.md:32]: Good point!  Making that edit and also adding a test to forbid "HTLC expiry delta" in the future.

**Comment by harding** [comment]
Edits made.  Thanks @LarryRuane @vostrnad @bitschmidty !

**Review by bitschmidty — APPROVED** [review]
Checked the diff, lgtm,

Thanks for putting these together @harding !

**Pull request merged**